Assets/AltTester/Runtime/AltDriver/Commands/AltSetStaticProperty.cs

```diff
@@ -1,4 +1,4 @@
-using AltTester.AltDriver.Commands;
+using AltTester.AltTesterUnitySDK.Driver.Commands;
 
 namespace Altom.AltDriver.Commands
```

Comment:
Is the namespace `Altom.AltDriver.Commands` correct, or should it be `AltTester.AltTesterUnitySDK.Driver.Commands` instead?

Reply:
it should be AltTester.AltTesterUnitySDK.Driver.Commands

Reply:
I will change it